openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -121,13 +122,20 @@ public void execute(ToolContext toolContext) {
             // Handle the code generation
             String oasFilePath = toolContext.filePath();
             Path packagePath = toolContext.currentPackage().project().sourceRoot();
-            Optional<OpenAPI> openAPI = getOpenAPIContract(packagePath, Path.of(oasFilePath), location, toolContext);
+            boolean isSanitized = false;
+            Map<String, ToolContext.Option> options = toolContext.options();
+            if (options != null && options.containsKey(IS_SANITIZED_OAS)) {
+                ToolContext.Option isUsingSanitizedOas = options.get(IS_SANITIZED_OAS);
+                String value = isUsingSanitizedOas.value().toString().trim();
```

Comment:
Since we define the option type as boolean, can't we just cast the value to boolean
```
boolean value = (boolean)isUsingSanitizedOas.value();
```